Katana/Plastic/Value.swift

```diff
@@ -145,3 +187,65 @@ public struct Value: Equatable {
     return lhs.scalable == rhs.scalable && lhs.fixed == rhs.fixed
   }
 }
+
+/**
+ This allows `Value` instances to be initialized by integer and floating point literals,
+ having the scalable part equal to the numeric literal, and the fixed part equal to 0.
+ */
+extension Value: ExpressibleByIntegerLiteral, ExpressibleByFloatLiteral {
```

Comment:
`Motivation: having a + operator between two heterogeneous types is more mysterious than..` but then why it is ok?
https://github.com/BendingSpoons/katana-swift/pull/76/files#diff-1c9939d1c3f55e132473f60fbc4c2739R100

I'm just trying to understand how to decide what is better and what is worse.

To me, already not being able to create a fully Value from an int is a signal that we shouldn't convert them. 

Plus try to think about a newcomers in katana/swift. It is better to explain that + is defined also for int and anchor or to say "you know.. numbers are automatically converted into value.. scalable [/fixed] value.. but not always. You need to look at the context and understand when it is converted"

Probably I'm not yet into the swift way of thinking, but the first explanation is way way more easy to understand.

PLUS: we are introducing this behaviour for a very specific case. Why should we do something that spread this behaviour everywhere when we can limit it (and limit the cases where this confusion may arise basically)
?